server/controllers/documents/download/helpers/decorateDocs.test.ts

```diff
@@ -259,6 +259,10 @@ describe('decorateDocs', () => {
           },
         ],
       },
+      {
```

Comment:
I don't think Note documents would appear in the uploaded documents array. The analogy is missing documents, which have their own array.

Reply:
ignore, this was wrong